src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContract.cs

```diff
@@ -699,8 +710,6 @@ private static DataContract CreateDataContract(int id, RuntimeTypeHandle typeHan
                                     dataContract = new CollectionDataContract(type);
                                 else if (type.GetTypeInfo().IsEnum)
                                     dataContract = new EnumDataContract(type);
-                                else if (Globals.TypeOfException.IsAssignableFrom(type))
```

Comment:
I presume this was removed because exceptions are now ISerializable? If that's the case, then this might be a breaking change. Someone targeting an earlier version of .Net Standard won't have ISerializable available so won't have a serialization constructor. What will happen if they now try to deserialize a previously serialized derived exception while running on the newer version? Will they have different behavior?